ee/billing/billing_manager.py

```diff
@@ -301,6 +301,7 @@ def update_org_details(self, organization: Organization, billing_status: Billing
                 events=usage_summary["events"],
                 recordings=usage_summary["recordings"],
                 rows_synced=usage_summary.get("rows_synced", {}),
+                feature_flag_requests=usage_summary.get("feature_flag_requests", {}),
```

Comment:
@zlwaterfield do I also need to add this entry to the billing manager, now that I'm quota limiting it?

Reply:
Yeah this will just check if the usage has changed and if the quotas should be checked.

Reply:
Okay, got it.  So I need to include this here?

Reply:
Yeah that looks correct